netdata.spec.in

```diff
@@ -47,18 +47,25 @@ make %{?_smp_mflags}
 rm -rf "%{buildroot}"
 make %{?_smp_mflags} install DESTDIR="%{buildroot}"
 find "%{buildroot}" -name .keep -exec rm {} \;
+touch %{buildroot}/%{_sysconfdir}/netdata/netdata.conf
```

Comment:
application should start with no file, if it is not the case we should fix the application.